tools/osx/crosstool/cc_toolchain_config.bzl

```diff
@@ -6228,6 +6250,7 @@ def _impl(ctx):
             supports_dynamic_linker_feature,
             objcopy_embed_flags_feature,
             dynamic_linking_mode_feature,
+            set_install_name,
```

Comment:
shouldn't this feature apply to dylibs built for iOS as well?

Reply:
Probably we should, done. PTAL